hedera-mirror-rest/api/v1/openapi.yml

```
@@ -162,6 +162,32 @@ paths:
           $ref: "#/components/responses/NotFoundError"
       tags:
         - accounts
+  /api/v1/accounts/{senderIdOrAliasOrEvmAddress}/airdrops/outstanding:
```

Believe this won't map properly unless we do
```suggestion
  /api/v1/accounts/{idOrAliasOrEvmAddress}/airdrops/outstanding:
```

---

Thanks I have updated this.